sklearn/tests/test_impute.py

```diff
@@ -614,17 +592,17 @@ def test_chained_imputer_missing_at_transform(strategy):
                   initial_imputer.transform(X_test)[:, 0])
 
 
-def test_chained_imputer_transform_stochasticity():
```

Comment:
This should be run with and without sample_posterior=True

Reply:
Will do.